### PR TITLE
[zh-cn]: update compatibility translation in 204 page

### LINKED ISSUE #23180: 204 状态码页浏览器兼容性无法正确显示
### MDN URL

https://developer.mozilla.org/zh-CN/docs/Web/HTTP/Status/204

### 与此议题相关的小节或标题？

浏览器兼容性

### 哪些信息是错误、无用或不完善的？

image
如图所示，浏览器兼容性无法正确渲染

### 你期望看到什么？

与别的状态码页面比较，别的页面上不存在浏览器兼容性小结，也许在204状态码的文档中也不应该写浏览器兼容性

### 你有什么支持的链接、参考或引用？

_No response_

### 你还想补充什么内容？

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `zh-cn/web/http/status/204`
* MDN URL: https://developer.mozilla.org/zh-CN/docs/Web/HTTP/Status/204
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/zh-cn/web/http/status/204/index.md
* Last commit: https://github.com/mdn/translated-content/commit/d322295b60e2ec47e0567937a7e439c849b19a9e
* Document last modified: 2022-09-21T23:46:25.000Z

</details>

### PR DESCRIPTION
**Pull request by MaxtuneLee**
### Description
remove browser compatibility section in 204 page

### Motivation
browser compatibility section is removed because it can not render correctly, and there is no browser compatibility section in other pages like [203 Non-Authoritative Information](https://developer.mozilla.org/zh-CN/docs/Web/HTTP/Status/203).

### Related issues and pull requests
https://github.com/mdn/translated-content/issues/23180

### CODE REVIEW

File: files/zh-cn/web/http/status/204/index.md [hunk 1]
```diff
@@ -19,10 +19,6 @@ HTTP **`204 No Content`** 成功状态响应码，表示该请求已经成功了
 
 {{Specifications}}
 
-## 浏览器兼容性
-
-{{Compat}}
-
```
Comment (yin1999) [inline files/zh-cn/web/http/status/204/index.md:25]: 请参考[英文文档的内容](https://github.com/mdn/content/blob/7f4dc4ea54b47cd6638d35b4456370b840125041/files/en-us/web/http/status/204/index.md?plain=1#L53-L60)进行更新

Comment (MaxtuneLee) [reply files/zh-cn/web/http/status/204/index.md]: 已根据其Compatibility note做了翻译更新👍🏻️

File: files/zh-cn/web/http/status/204/index.md [hunk 2]
```diff
@@ -19,9 +19,11 @@ HTTP **`204 No Content`** 成功状态响应码，表示该请求已经成功了
 
 {{Specifications}}
 
-## 浏览器兼容性
+## 兼容性说明
 
-{{Compat}}
+尽管此状态码适用于没有响应体的响应，但服务器可能会错误地在标头后面携带数据。这种情况在长连接中尤为常见，因为无效的响应体可能会与后续请求的响应数据混合在一起。HTTP 协议允许浏览器以不同的方式处理此类响应（[HTTPWG `http-core` GitHub 存储库](https://github.com/httpwg/http-core/issues/26) 中正在持续讨论有关规范的内容）。
```
Comment (yin1999) [inline files/zh-cn/web/http/status/204/index.md:24]:
```suggestion
尽管此状态码适用于没有响应体的响应，但服务器可能会错误地在标头后面携带数据。这种问题在长连接中尤为常见，因为无效的响应体可能会与后续请求的响应数据混合在一起。HTTP 协议允许浏览器以不同的方式处理此类响应（[HTTPWG `http-core` GitHub 仓库](https://github.com/httpwg/http-core/issues/26)中正在持续讨论有关规范文本的内容）。
```

**Review by yin1999 — APPROVED** [review]
Thanks

**Comment by yin1999** [comment]
Welcome to the community @MaxtuneLee :tada:

**Pull request merged**